src/main/resources/templates/storefront_management.html

```diff
@@ -1,8 +1,7 @@
 <!DOCTYPE html>
 <html xmlns:th="http://www.thymeleaf.org" lang="en">
 <html>
-<head>
-    <head th:replace="fragments/header :: header"></head>
+<head th:replace="fragments/header :: header"></head>
```

Comment:
Removed unnecessary nested <head> tags to simplify the HTML structure, enhancing readability and maintainability. This cleanup aligns with clean code principles discussed in ["Java Clean Code"](https://www.baeldung.com/java-clean-code), which recommend avoiding redundant elements that clutter code without adding value.